addons/pharma/functions/fnc_getBloodVolumeChange.sqf

```diff
@@ -62,20 +72,32 @@ if (!isNil {_unit getVariable [QACEGVAR(medical,ivBags),[]]}) then {
                 };
             };
 
-            // Plasma adds to ECP. Saline splits between the ECP and ISP. Blood adds to ECB
+            // Plasma adds to ECP. Saline splits between the ECP and ISP. Blood splits between the ECP and ECB
             switch (true) do {
                 case(_type == "Plasma"): { _ECP = _ECP + _bagChange; _lossVolumeChange = _lossVolumeChange + (_bagChange / ML_TO_LITERS); };
-                case(_type == "Saline"): { 
+                case(_type in ["Saline", "RingersLactate"]): { 
```

Comment:
Ringers should not be pushing as much into ISP because it has the same osmotic pressure as blood. Make it three to one ECP to ISP